localstack/services/kms/kms_listener.py

```diff
@@ -285,17 +279,28 @@ def _generate_data_key_pair(data, create_cipher=True):
     if create_cipher:
         cipher_text = kms.encrypt(KeyId=key_id, Plaintext=private_key)["CiphertextBlob"]
         cipher_text_blob = base64.b64encode(cipher_text)
+
+    region = region_details.get_current_request_region()
     result = {
         "PrivateKeyCiphertextBlob": cipher_text_blob,
         "PrivateKeyPlaintext": base64.b64encode(private_key),
         "PublicKey": base64.b64encode(public_key),
         "KeyId": key_id,
         "KeyPairSpec": key_spec,
         "KeySpec": key_spec,
+        "KeyUsage": "SIGN_VERIFY",
+        "Policy": data.get("Policy"),
+        "Region": region,
+        "Description": data.get("Description"),
+        "Arn": "arn:aws:kms:%s:000000000000:key/%s" % (region, key_id),
```

Comment:
Let's better use the `kms_key_arn()` util function from `aws_stack.py` here. (We should avoid hardcoding the account ID `000000000000`, as it may change over time)